packages/sdk/app-framework/src/playground/generator/generator.ts

```diff
@@ -2,14 +2,27 @@
 // Copyright 2025 DXOS.org
 //
 
+import { S } from '@dxos/echo-schema';
+
+import { Capabilities, Events } from '../../common';
 import { contributes, defineEvent, defineCapability, defineModule, definePlugin } from '../../core';
+import { createResolver, type IntentSchema } from '../../plugin-intent';
 
 export const Number = defineCapability<number>('dxos.org/test/generator/number');
 
 export const CountEvent = defineEvent('dxos.org/test/generator/count');
 
 export const createPluginId = (id: string) => `dxos.org/test/generator/${id}`;
 
+export const createGeneratorIntent = (id: string) => {
```

Comment:
What does this do? Can you explain the Alert section of this?

Reply:
it's an intent which window.alerts the generated capability, was just using this to test that intents could be added dynamically